src/utils/cron.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-useless-path-segments */
+/* eslint-disable node/no-missing-require */
+/* eslint-disable import/no-unresolved */
+
+const mongoose = require('mongoose');
+const cron = require('node-cron');
+const sendEmail = require('../utils/mailer');
+const logger = require('../utils/logger');
```

Comment:
Hi @Kishimoto96. These eslint issues comes from following reasons:
- You go up one directory and go back in to the same directory. Instead of `../utils/mailer` you should do `./mailer`.
- You don't have node-cron installed in your dependencies. Try to `yarn add node-cron` and then run `yarn`. 
I think these changes will solve the errors.